docs/data/charts/radar/radar.md

```diff
@@ -53,6 +53,18 @@ The radar chart displays a grid behind the series that can be configured with:
 
 {{"demo": "DemoRadar.js" }}
 
+## Axis values
+
+You can add labels to metrics with the `<RadarAxis />`.
+This component requires a `metric` prop and can be configured with:
+
+- `angle` The angle used to display labels. By default it's the one associated to the given metric.
+- `labelOrientation` The orientation strategy. Either horizontal labels with moving anchor point, or label rotating with the axis.
+- `divisions` The number of labels to display.
```

Comment:
Why is this different than `<RadarChart divisions/>`? Maybe also remove the "sync" in the example, as that is confusing, as changing the RadaAxis divisions shouldnt affect the grid.

Reply:
My point was more to show them that you can have different values between them. I was confused at first when I saw that the axis had its own "divisions". It makes sense that they are separate, but the example should threat them as separate things so it is clear to user.